### ISSUE TITLE
NicheNet Seurat Wrapper on integrated data

### ISSUE THREAD

**Issue by dmitrikotov** [saeyslab/nichenetr#63]
Thank you for the wonderful package! I have been running NicheNet on integrated Seurat datasets and it has worked wonderfully when I ran NicheNet step by step. Unfortunately, I am having issues running the NicheNet seurat wrapper (nichenet_seuratobj_aggregate) on integrated data. It seems like I can't use the seurat object's RNA data in place of the integrated data, because I am unable to provide the assay_oi argument to the nichenet_seuratobj_aggreate function. Are you planning on including integrated dataset functionally into the nichenet_seuratobj_aggreate function in the future?

**Comment by kc-mcp** [comment]
I am also having the same issue and have not found a solution yet.

**Comment by browaeysrobin** [comment]
Hi @dmitrikotov and @kc-mcp 

Thank you for pointing this out. I worked on it and normally it should work now after re-installing nichenetr for those 3 wrapper functions: `nichenet_seuratobj_aggregate`, `nichenet_seuratobj_aggregate_cluster_de`,  `nichenet_seuratobj_cluster_de`. 
Can you let me know whether it works?

**Comment by dmitrikotov** [comment]
Thank you for updating the package. I tried out nichenet_seuratobj_aggregate and included assay_oi = "RNA" as an argument. I did not get an error for including this argument, so it seems like your update has worked. However, I have a new error. Using Seurat V4, the FindMarkers function generates a column called avg_log2FC. It seems that nichenet_seuratobj_aggregate filters results from the FindMarkers function (I believe when it uses the get_lfc_celltype function?), but the expected column for filtering is avg_logFC rather than the actual label avg_log2FC which results in the error. Would it be possible to update the nichenet_seuratobj_aggregate function to expect avg_log2FC in place of or in addition to avg_logFC? Thank you!
